en/SUMMARY.md

```diff
@@ -71,6 +72,7 @@
   * [Drivers](middleware/modules_driver.md)
     * [Distance Sensor](middleware/modules_driver_distance_sensor.md)
   * [Estimators](middleware/modules_estimator.md)
+  * [Simulations](middleware/modules_simulation.md)
```

Comment:
This summary page change is good/needed. The file itself gets generated and copied in place for you.